src/api/wallet/WalletApiImpl.ts

```diff
@@ -4,6 +4,104 @@ import assert from 'assert'
 import { getDefaultProvider } from '../'
 
 import WalletConnectProvider from '@walletconnect/web3-provider'
+// import WalletConnectQRCodeModal from '@walletconnect/qrcode-modal'
+;(window as any).WalletConnectProvider = WalletConnectProvider
+
+const oldEnable = WalletConnectProvider.prototype.enable
```

Comment:
I would be OK keeping the workaround, but we need an issue to remove it, and also open an issue to WC.

If this works I'm ok with it. Testing now.